test1.js

```diff
@@ -3,3 +3,9 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+var okey = '' ;
+
+if(okey    === "okey") {
```

Comment:
Codacy has a fix for the issue: [Replace `(okey···` with `·(okey`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5798)
```suggestion
if (okey === "okey") {
```